src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -75,6 +75,8 @@ public class MicrobeEditor : NodeWithInput, ILoadableGameState, IGodotEarlyNodeR
     [JsonProperty]
     private float rigidity;
 
+    private float initialRigidity;
```

Comment:
I really dislike doing a special case for initialRigidity. If this MP compare to initial rigidity is wanted, the full dynamic recompute approach should be done at once. There's an abandoned PR about that: https://github.com/Revolutionary-Games/Thrive/pull/2365

Reply:
I know. But it is abandoned; and I cannot think of better solutions.

Reply:
I'll delete this then.

Reply:
I was suggesting that if you are willing to pick it up, you could continue that PR.

Reply:
Well, tried, failed. Too many problems there. Some out of my talent...

Reply:
Maybe someone else on the team could help with that? From what I remember that PR wasn't far off from being finished, just a little bit of refactoring to separate the cost and actual applying of an action functions.

Reply:
It failed to handle overlaps (place sth on a cytoplasm costs a place *and* a delete; and if I place another cytoplasm elsewhere then it costs a move. I find myself unable to handle that. Maybe I should think harder.)